server/src/main/java/org/elasticsearch/rest/action/search/RestSearchAction.java

```diff
@@ -308,6 +309,11 @@ static void preparePointInTime(SearchRequest request, NamedWriteableRegistry nam
         if (request.preference() != null) {
             validationException = addValidationError("[preference] cannot be used with point in time", validationException);
         }
+        if (restRequest.hasParam("ccs_minimize_roundtrips")) {
```

Comment:
I think we should still accept the param if it is set to false ?

Reply:
Fixed in https://github.com/elastic/elasticsearch/pull/61827/commits/44f14d3785db37de1fa33ec197e3501523d220de